docs/source/nodeoperations/restore.md

```diff
@@ -2,88 +2,223 @@
+```console
+sctool restore -c <CLUSTER_ID> -L <BACKUP_LOCATION> -T <SNAPSHOT_TAG> --restore-tables
 ```
 
-We will be using `sstableloader` to restore data. `sstableloader` needs a specific directory structure to work namely: `<keyspace>/<table>/<contents>`
-To create this directory structure and download all the files execute these commands:
-```bash
-mkdir snapshot
-cd snapshot
-# Create temporary directory structure.
-cat ../backup_files.out | awk '{print $2}' | xargs mkdir -p
-# Download snapshot files.
-cat ../backup_files.out | xargs -n2 aws s3 cp
+Where:
+* `CLUSTER_ID` - is a name or ID of a cluster we are restoring to.
+* `BACKUP_LOCATION` - location where backup is stored.
+* `SNAPSHOT_TAG` - snapshot tag of a backup we are restoring from.
+
+After task is created, output of the command will return ID of a restore task.
+```console
+$ sctool restore -c scylla/target -L s3:source-backup -T sm_20240105115931UTC --restore-tables
+restore/63642069-bed5-4def-ba0f-68c49e47ace1
 ```
 
-To load data into cluster pass cluster address to `sstableloader` together with path to data files and credentials:
-```bash
-sstableloader -d 'simple-cluster-us-east-1-us-east-1a-0.scylla.svc,simple-cluster-us-east-1-us-east-1a-1.scylla.svc,simple-cluster-us-east-1-us-east-1a-2.scylla.svc' ./users/data_0 --username scylla --password <password>
+To check progress use:
+```console
+$ sctool progress -c scylla/target restore/63642069-bed5-4def-ba0f-68c49e47ace1
+Restore progress
+Run:            ab015cef-abc8-11ee-9521-6e7993cf42ed
+Status:         DONE
+Start time:     05 Jan 24 12:48:04 UTC
+End time:       05 Jan 24 12:48:15 UTC
+Duration:       11s
+Progress:       100% | 100%
+Snapshot Tag:   sm_20240105115931UTC
+
++-------------+-------------+--------+---------+------------+--------+
+| Keyspace    |    Progress |   Size | Success | Downloaded | Failed |
++-------------+-------------+--------+---------+------------+--------+
+| users       | 100% | 100% | 409MiB |  409MiB |     409MiB |      0 |
++-------------+-------------+--------+---------+------------+--------+
+
+Post-restore repair progress
+Run:            ab015cef-abc8-11ee-9521-6e7993cf42ed
+Status:         DONE
+Start time:     05 Jan 24 12:48:04 UTC
+End time:       05 Jan 24 12:48:15 UTC
+Duration:       11s
+Progress:       100%
+Intensity:      1
+Parallel:       0
+Datacenters:
+  - us-east-1
+
++-------------+--------------+----------+----------+
+| Keyspace    |        Table | Progress | Duration |
++-------------+--------------+----------+----------+
+| users       | users        | 100%     | 0s       |
++-------------+--------------+----------+----------+
+
 ```
 
-Depending on how big is your data set, this operation may take some time.
-Once it finishes, data from the snapshot is restored and you may clean up the host.
+As last step, we need to manually reset `tombstone_gc` via `cqlsh` in target ScyllaCluster on **every table and every keyspace** which was restored.
```

Comment:
```suggestion
When the restore procedure has finished, you will need to manually reset `tombstone_gc` via `cqlsh` in target ScyllaCluster on **every table and every keyspace** which was restored.
```

Reply:
"last step" is not going to age well if we ever extend it

Reply:
done